auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -601,6 +847,38 @@ export default defineComponent({
       border-top-right-radius: 0;
       border-bottom-right-radius: 0;
       margin-right: 1px;
+      max-height: 36px !important;
+      min-height: 36px !important;
```

Comment:
This is based on another uxpin I got from Janis. I also add it in the ticket.
https://preview.uxpin.com/a86dccdd20ac62828f965c5eea10f81b828a9853#/pages/163836595/simulate/sitemap